spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfigurationTests.java

```diff
@@ -107,6 +111,12 @@ public void testDatastoreHealthIndicatorNotCreated() {
 				.isThrownBy(() -> context.getBean(DatastoreHealthIndicator.class)));
 	}
 
+	private Datastore getDatastoreBean(ApplicationContext context) {
```

Comment:
Maybe this method should be in `DatastoreTemplate` for convenience, or is it too risky?

Reply:
This is just for tests actually (this is a test class)

Reply:
Yeah, I'm just suggesting to add this utility method to `DatastoreTemplate`.

Reply:
Oh... For what purpose?

Reply:
To avoid dealing with the provider.

Reply:
I think it's ok. It's hard to misuse the supplier because it is of the wrong type unless you call .get() on it, which is the only method it has.